portal/templates/admin/patients_by_org.html

```diff
@@ -35,6 +35,7 @@ <h4 class="tnth-headline">{% trans %}Patient List{% endtrans %}</h4>
              data-toggle="table"
              data-search="true"
              data-pagination="true"
+             {% if 'status' in config.PATIENT_LIST_ADDL_FIELDS %}data-report-export="true"{% endif %}
```

Comment:
Might be nice to move that entirely to the frontend :)

Reply:
done, thanks ivan